lib/logger/lib/logger/config.ex

```diff
@@ -284,7 +305,19 @@ defmodule Logger.Config do
     end
   end
 
+  defp read_data(key) do
+    if :erlang.module_loaded(:persistent_term) do
```

Comment:
Have you checked if maybe `function_exported?` will be faster? The module will always be loaded if available since it's a preloaded module.